docker/build_image.sh

```diff
@@ -22,7 +22,7 @@ do
           echo "-g, --gpu specify to use gpu"
           echo "-bt, --buildtype specify to created image for codebuild. Possible values: production, dev, codebuild."
           echo "-cv, --cudaversion specify to cuda version to use"
-          echo "-ub, --ubuntu specify ubuntu version. Possible values: ubuntu:18.04, ubuntu 20.04"
+          echo "-ub, --ubuntu specify ubuntu version. Possible values: ubuntu:20.04, ubuntu 22.04"
```

Comment:
Wondering if we have done any testing with 22.04 if are adding it here as a possible option

Reply:
Let's kill the Ubuntu flag and make a separate PR for this later

Reply:
Hello @msaroufim @maaquib I removed the reference to ubuntu 22.04 from the script.